docs/product/issues/issue-details/index.mdx

```diff
@@ -102,6 +102,36 @@ If you’ve enabled [Session Replay](/product/explore/session-replay/), you’ll
 
 ![Session Replay](./img/issue-replay.png)
 
+## Feature Flags
+There are two types of integrations you can configure to integrate feature flag data into your error events.
+
+The first type is flag evaluation tracking within the Sentry SDK. To enable it, update the Sentry SDK's "init" method to include one of our supported feature flag integrations. Support varies by language and by provider. Doing so will give you access to the feature flag table which shows any flags that were evaluated after the Sentry SDK's initialization and before the error event occurred. You will see rows of flag, evaluation-result pairs.
+
+The second type is feature flag change tracking. This will enable a timeline of feature flag definition changes inside the releases graph. These annotations can help you diagnose when a feature flag change is related to a new issue. To enable this type of integration, please visit the documentation page of one of our supported providers:
```

Comment:
```suggestion
The second type is **feature flag change tracking**. This will enable a timeline of feature flag definition changes on the event and user volume chart on the Issue Details page. These annotations can help you diagnose when a feature flag change is related to a new issue. To enable this type of integration, please visit the documentation page of one of our supported providers:
```